lib/embed/stacks_image.rb

```diff
@@ -0,0 +1,24 @@
+module Embed
+  ###
+  # A module to be mixed into classes to get generate
+  # URLs to stacks images using the IIIF image API
```

Comment:
👍 Good 👀 

I'll fix this as soon as I'm at a good stashing point in my current working branch.